packages/back-end/src/controllers/features.ts

```diff
@@ -535,6 +538,34 @@ export async function deleteFeatureById(
   });
 }
 
+export async function postFeatureArchive(
+  req: AuthRequest<null, { id: string }>,
+  res: Response
+) {
+  req.checkPermissions("createFeatures", "publishFeatures");
+  const { id } = req.params;
+  const { org } = getOrgFromReq(req);
+  const feature = await getFeature(org.id, id);
+
+  if (!feature) {
+    throw new Error("Could not find feature");
+  }
+  await archiveFeature(feature.organization, id, !feature.archived);
+
+  await req.audit({
+    event: "feature.archive",
+    entity: {
+      object: "feature",
+      id: feature.id,
+    },
```

Comment:
Should include details here so when someone looks at the audit entry, they'll know if the feature was archived or unarchived.

```ts

    details: auditDetailsUpdate(
      { archived: feature.archived }, // Old state
      { archived: !feature.archived } // New state
    ),
```